### PR TITLE
Update from master.

### PR DESCRIPTION
**Pull request by nicolasnoble**
Tweaking binding.gyp.template to avoid properties starting with a $.

### CODE REVIEW

File: packages/grpc-native-core/binding.gyp [hunk 1]
```diff
@@ -194,25 +204,45 @@
...
+              '-Ithird_party/upb',
+              '-Isrc/core/ext/upb-generated',
               '-fvisibility=hidden',
           ],
           'OTHER_CPLUSPLUSFLAGS': [
               '-g',
               '-Wall',
               '-Wextra',
               '-Werror',
+              '$(W_NO_UNKNOWN_WARNING_OPTION)',
```
Comment (murgatroid99) [inline packages/grpc-native-core/binding.gyp]: Is this supposed to be here?

Comment (nicolasnoble) [reply packages/grpc-native-core/binding.gyp]: Gah, I'm not properly filtering all of them out. Thanks.

Comment (nicolasnoble) [reply packages/grpc-native-core/binding.gyp]: Fixed.

**Comment by murgatroid99** [comment]
It looks like the Mac builds are failing because XCode doesn't recognize `-Wno-maybe-uninitialized`. Based on the error message it looks like that might be resolved by passing `-Wno-unknown-warning-option`.

**Comment by murgatroid99** [comment]
This has been superseded by more recent updates.

**Pull request closed without merging**